app/templates/components/talent-report.hbs

```diff
@@ -0,0 +1,38 @@
+<div class="panel panel-default panel-ews talent-report">
+  <!-- Default panel contents -->
+  <div class="panel-heading">
+    Talents assigned to you (as TM)
+    <span class="hidden-xs subtitle">
+      {{if displaySubtitle "Click the staff name or status to intervene with attrition evaluation"}}
+    </span>
+  </div>
+
+  <!-- Table -->
+  <table class="table">
+    <thead>
+      <tr>
+        <th class="col-sm-1">Pic</th>
+        <th class="col-sm-2">Name</th>
+        <th class="col-sm-1">Employee#</th>
+        <th class="col-sm-2">Position</th>
+        <th class="col-sm-2">Last Evaluated</th>
+        <th class="col-sm-3">Current Rate / Parameter</th>
+        <th class="col-sm-1">&nbsp;</th>
+      </tr>
+    </thead>
+    <tbody>
+      {{#each reports as |report|}}
+      <tr>
+        <td><img src={{report.photo}}></td>
+        <td><strong>{{report.fullName}}</strong></td>
+        <td>{{report.employeeNo}}</td>
+        <td>{{report.position}}</td>
+        <td>{{moment-format (now) 'DD MMMM YYYY'}}</td>
+        <td class="rate{{report.rateNo}}"><strong>{{report.rateText}}</strong></td>
+        <td>{{#if report.intervene}}<a class="intervene" onclick="alert('Email sent!')"><strong>I'll Intervene</strong></a>{{/if}}</td>
```

Comment:
we'll use link-to helper here in the near future when we implement our modal.. for now we can leave it blank :)